index.html

```diff
@@ -106,17 +106,19 @@ <h3>Stock performance data</h3>
         </section>
         <section id="section-three">
             <div class="row newshead" id="news">
-                <div class="column" id="newshead">
+                <div class="column" id="newshead">   
                     <h3>Latest news about <span>[populate name/industry/sector from the input form]</span></h3>
                 </div>
             </div>
             <div class="row articles">
-                <article class="column column-80 news-item" id="item01"><br><br><br><br>Link populated from ajax request<br><br><br><br></article>
-                <article class="column column-80 news-item" id="item02"><br><br><br><br>Link populated from ajax request<br><br><br><br></article>
-                <article class="column column-80 news-item" id="item03"><br><br><br><br>Link populated from ajax request<br><br><br><br></article>
-                <article class="column column-80 news-item" id="item04"><br><br><br><br>Link populated from ajax request<br><br><br><br></article>
-                <article class="column column-80 news-item" id="item05"><br><br><br><br>Link populated from ajax request<br><br><br><br></article>
-                <article class="column column-80 news-item" id="item06"><br><br><br><br>Link populated from ajax request<br><br><br><br></article>
+                <article class="column column-80 news-item" id="item00"><br><br><br><br></article>
+                <article class="column column-80 news-item" id="item01"><br><br><br><br></article>
+                <article class="column column-80 news-item" id="item02"><br><br><br><br></article>
+                <article class="column column-80 news-item" id="item03"><br><br><br><br></article>
```

Comment:
Maybe also remove the 4 <br> between the <article> tags. That was just a placeholder for the mock-up. Let's see how it loads and then make the styling changes necessary.